contracts/App.sol

```diff
@@ -20,7 +20,7 @@ contract App is AragonApp {
     ////               ....
     ////
     ////  Build something beautiful.
```

Comment:
We should change the `pragma solidity` declaration at the top to be `^0.4.24` instead.